openverse_catalog/dags/commoncrawl/commoncrawl_etl.py

```diff
@@ -169,12 +169,13 @@
 )
 
 with dag:
-    check_for_cc_index = S3PrefixSensor(
+    check_for_cc_index = S3KeySensor(
         task_id="check_for_cc_index",
         retries=0,
         aws_conn_id=AWS_CONN_ID,
         bucket_name=COMMONCRAWL_BUCKET,
-        prefix=f"crawl-data/{CC_INDEX_TEMPLATE}",
+        bucket_key=f"crawl-data/{CC_INDEX_TEMPLATE}*",
+        wildcard_match=True,
```

Comment:
> wildcard_match: whether the bucket_key should be interpreted as a Unix wildcard pattern

I see the `*` added to make this work. why do we need it? More specifically why wouldn't `wildcard_match=False` without the astrix work?

Reply:
I believe we want `wildcard_match` so that the sensor sees this key as a potential prefix and uses the list objects API rather than the head object one (described here: https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/operators/s3.html#wait-on-an-amazon-s3-key)